test/functional/bumpfee.py

```diff
@@ -136,6 +138,21 @@ def test_nonrbf_bumpfee_fails(peer_node, dest_address):
     assert_raises_rpc_error(-4, "not BIP 125 replaceable", peer_node.bumpfee, not_rbfid)
 
 
+def test_nonrbf_bumpfee_fails_then_succeeds(test, dest_address):
+    test.stop_nodes()
+    test.start_node(0, extra_args=["-prematurewitness", "-walletprematurewitness", "-mempoolreplacementtimeout=30"])
```

Comment:
Why stop `node[1]`?

If not, remove`stop_nodes` above and here can be:
```py
test.restart_node(0, ["-prematurewitness", "-walletprematurewitness", "-mempoolreplacementtimeout=30"])
```
Use `restart_node` below.